manifests/webhook.pp

```diff
@@ -61,11 +61,31 @@
     before => File['webhook_init_script'],
   }
 
+  if $service_file =~ /systemd/ {
+
+    $_service_file_mode = '0644'
+
+    exec { 'webhook_systemd_reload':
```

Comment:
I don't mind, but referred module has a design flaw, and was addressed for more then a year, as explained here https://github.com/camptocamp/puppet-systemd/issues/5
Since authors are leaning to change class to 'define', it will be essentially the same implementation